Server/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -1204,6 +1215,24 @@ bool CStaticFunctionDefinitions::SetElementPosition(CElement* pElement, const CV
     return true;
 }
 
+bool CStaticFunctionDefinitions::SetDummyRotation(CElement* pElement, const CVector& vecRotation)
```

Comment:
Is this even used anywhere?

Reply:
Used in SetElementRotation 
https://github.com/multitheftauto/mtasa-blue/blob/0dfac0a08005fd053d8b56fddf0a8ce047d6b859/Server/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp#L1285

https://github.com/multitheftauto/mtasa-blue/blob/0dfac0a08005fd053d8b56fddf0a8ce047d6b859/Server/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp#L1290